src/main/dialogs/aboutDialog.js

```diff
@@ -34,7 +34,7 @@ fluid.defaults("gpii.app.aboutDialog", {
     config: {
         params: {
             userListeners: ["USB", "NFC", "Fingerprint", "Webcam & Voice"],
-            version: { expander: { func: app.getVersion } }
+            version: { expander: { func: "gpii.app.electron.app.getVersion" } }
```

Comment:
This looks weird and hard to read. Any particular reason for making this change?